src/stratis_cli/_parser/_pool.py

```diff
@@ -37,6 +37,15 @@
                         help="Redundancy level for this pool",
                     ),
                 ),
+                (
+                    "--keyfile",
+                    dict(
+                        default=None,
+                        type=str,
+                        help="Path to keyfile for optional encryption of a pool",
```

Comment:
Can you drop the word "optional" since ```--keyfile``` is already optional?

Reply:
Resolved.